docs/code/creating-and-writing-nwbfile.py

```diff
@@ -109,6 +109,36 @@ def main():
     f.add_acquisition(spatial_ts, [ep1, ep2])
     # create-timeseries: end
 
+    # create-compressed-timeseries: start
+    from pynwb.ecephys import ElectricalSeries
+    from pynwb.behavior import SpatialSeries
+    from pynwb.form.backends.hdf5 import H5DataIO
+
+    ephys_ts = ElectricalSeries('test_ephys_data',
```

Comment:
This script gets executed as a test. You'll need to rename these so that it runs